source/fuzz/transformation_record_synonymous_constants.cpp

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2020 Stefano Milizia
+// Modifications Copyright (c) 2020 Google LLC
```

Comment:
I think this will fail the project's formatting checks, and also you typically just list all the copyright holders - you don't tend to say "Modifications".

Reply:
Thanks, I was unsure about how to do this so I looked in the repository and found this way. Should I change it? Do I list all the copyright holders just by putting a comma between them?

Reply:
(for example in source/operand.cpp)

Reply:
Fixed